documentation/operators/src/nodes/mix-node-setup.md

```diff
@@ -0,0 +1,315 @@
+
+#### Bond via the CLI (power users)
+If you want to bond your mix node via the CLI, then check out the [relevant section in the Nym CLI](https://nymtech.net/docs/tools/nym-cli.html#bond-a-mix-node) docs.
+
+### Running your mix node
+
+Now you've bonded your mix node, run it with:
+
+```
+./nym-mixnode run --id <YOUR_ID>
+```
+
+If everything worked, you'll see your node running on the either the [Sandbox testnet network explorer](https://sandbox-explorer.nymtech.net) or the [mainnet network explorer](https://explorer.nymtech.net), depending on which network you're running.
+
+Note that your node's public identity key is displayed during startup, you can use it to identify your node in the list.
+
+<!---serinko - propose to drop this sentence:
+
+Keep reading to find out more about configuration options or troubleshooting if you're having issues. There are also some tips for running on AWS and other cloud providers, some of which require minor additional setup.
+
+--->
+
+Have a look at the saved configuration files in `$HOME/.nym/mixnodes/` to see more configuration options.
+
+## Node Description (optional)
+
+In order to easily identify your node via human-readable information later on in the development of the testnet when delegated staking is implemented, you can `describe` your mixnode with the following command:
+
+```
+./nym-mixnode describe --id <YOUR_ID>
+```
+Node description is a short text that describes your node. It is displayed in the `./nym-mixnode list` command and in the `./nym-mixnode node-details --id <YOUR_ID>` command. It also shows up in the node explorer to let people know what your node is about and link to your website.
+
+You can set your node description, by creating a file called `description.toml` and put it in the same directory as your `config.toml` file (`~/.nym/mixnodes/<YOUR_ID>/description.toml`). The file should look like this example:
+
+```toml
+name = "Winston Smith"
+description = "I am the Sphinx"
+link = "https://nymtech.net"
+location = "Giza, Egypt"
+```
+
+> Remember to restart your mix node process in order for the new description to be propagated.
+
+## Node Families
+
+Node family involves setting up a group of mix nodes that work together to provide greater privacy and security for network communications. This is achieved by having the nodes in the family share information and routes, creating a decentralized network that makes it difficult for third parties to monitor or track communication traffic.
+
+### Create a Node Family
+
+To create a Node family, you will need to install and configure multiple mix nodes, and then use the CLI to link them together into a family. Once your Node family is up and running, you can use it to route your network traffic through a series of nodes, obscuring the original source and destination of the communication.
+
+You can use either `nym-cli` which can be downloaded from the [release page](https://github.com/nymtech/nym/releases) or compiling `nyxd`.
```

Comment:
`nym-cli` is in `tools/nym-cli` - I'd link to `nym-cli` page on the docs as well here